api/cmd/api/ui_handler.go

```diff
@@ -22,6 +22,10 @@ type uiEnvHandler struct {
 	AlertEnabled bool `json:"REACT_APP_ALERT_ENABLED"`
 
 	ModelDeletionEnabled bool `json:"REACT_APP_MODEL_DELETION_ENABLED"`
+
+	ImageBuilderCluster   string `json:"REACT_APP_IMAGE_BUILDER_CLUSTER"`
+	ImageBuilderProject   string `json:"REACT_APP_IMAGE_BUILDER_PROJECT"`
+	ImageBuilderNamespace string `json:"REACT_APP_IMAGE_BUILDER_NAMESPACE"`
```

Comment:
im not too sure to use an `ImageBuilderConfig` instead of string , it has way too many config than required